recipes/qt/5.x.x/conanfile.py

```diff
@@ -926,7 +926,7 @@ def package_info(self):
         libsuffix = ""
         if not self.options.multiconfiguration:
             if self.settings.build_type == "Debug":
-                if self.settings.os == "Windows":
+                if self.settings.os == "Windows" and self._is_msvc:
```

Comment:
there's a new tool helper which should be able to replace this property https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc

Reply:
Should there be a version check since this was changed during 5.x?

Reply:
The tool to check about MSVC is interesting, thank you.
Because `self._is_msvc` is used all over the place, I did not use it here. Should that be another PR ?

For the version check: yes, I think you are right.
By looking [here](https://codereview.qt-project.org/c/qt/qtbase/+/268290): I understand that the first version is 5.14.0.
I just updated the recipe, could you take a look ?

Reply:
I try to build Qt5 with this patch (examples of builds are available [here](https://gitlab.com/scandyna/conan-center-index-partialbasicci-sandbox/-/pipelines/630739776)).

For Qt 5.13, it always fails:
```shell
ERROR: qt/5.13.2: Error in configure() method, line 268
	assert config.sections()
	AssertionError: 
```
If I understand, `qtmodules5.13.2.conf` does not exist. Looking at the folder, it seems that only Qt 5.15 is supported.

Also looking [here](https://endoflife.date/qt), it also seems that Qt5 < 5.15 are end of life versions.

If the above is true, I think that the version check should be removed (it adds unnecessary complexity).

Reply:
Yes!

Reply:
I removed the Qt version check.

Reply:
> Should that be another PR ?

It could be. But that would take up more resources and be another review.
A simple search and replace seems reasonable to sneak in here.

But that's your choice I don't mind either way